app/lib/clients/vault/key_value.rb

```diff
@@ -36,21 +51,34 @@ def kv_engine_type
         "kv-v2"
       end
 
-      def create_kv_policy(path)
-        client.sys.put_policy(policy_path(path), kv_policy(path))
+      def create_kv_policies(path)
+        client.sys.put_policy(producer_policy_path(path), kv_producer_policy(path))
+        client.sys.put_policy(consumer_policy_path(path), kv_consumer_policy(path))
       end
 
-      def policy_path(path)
-        "kv_policy/#{path}"
+      def producer_policy_path(path)
+        "kv_policy/#{path}/producer"
```

Comment:
"kv_policy/" seems a bit generic.  Do we need to call it "astral_kv_policy" to be safe?

Reply:
uses the `kv_mount` (kv_astral) var to form this now.